apigee_edge_ui.module

```diff
@@ -0,0 +1,170 @@
+<?php
+
+/**
+ * @file
+ * Apigee Edge UI module for Drupal.
+ *
+ * Copyright (C) 2019 PRONOVIX GROUP BVBA.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301,
+ * USA.
+ */
+
+/**
+ * @file
+ * Main module file for Apigee Edge UI.
```

Comment:
Try to avoid double `@file` header.

Reply:
@kittiradovics I would have removed this one and kept the `@file` annotation on the top. According to CS a php file should start with that. "Apigee Edge UI module for Drupal." already describes what is this file.